### ISSUE TITLE
http-stats attribute is not honoring the status settings

### ISSUE THREAD

**Issue by ttrading** [VerizonDigital/vflow#148]
We installed v0.9 on a CentOS system.

http-stats: false is listed in the vflow.conf. However, when vflow is started it is still trying to listen on port TCP8081.

I am thinking this is due to the fact that ipv6 is enabled on this server but I just wanted to check with you first.

Thanks,

**Comment by mehrdadrad** [comment]
@ttrading pls try stats-enabled: false instead.
https://github.com/VerizonDigital/vflow/blob/master/docs/config.md

**Comment by ttrading** [comment]
Sorry for the typo.. Yes, it is set to "stats-enabled: false".

The way we stumbled upon this issues was that we had another service running on TCP8081. After a reboot, vflow grabbed the port before that other process and started to cause issues.

We confirmed that stats-enabled is false but the vflow process was disregarding it and listening on port 8081 so as a workaround we changed the listen port to 8071. 

However, I figured it is still worth sharing this with you.

Thanks,
